### ISSUE TITLE
Truyenqq devtool detected

### ISSUE THREAD

**Issue by Karutovn** [keiyoushi/extensions-source#3262]
### Source information

Truyenqq 1.4.14

### Source language

Vietnamese 

### Steps to reproduce

Open source just like this video 

https://github.com/keiyoushi/extensions-source/assets/73945551/6afa808f-853c-4e5e-9f78-13221ca9c4c8

### Expected behavior

The manga should by found and devtool should be bypassed

### Actual behavior

This happend instead open anything in this source

### Mihon/Tachiyomi version

Tachiyomij2k 1.7.4

### Android version

Android 14 (one ui 6.0)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by MikeZeDev** [comment]
I just tested this workaround on Hakuneko, may work on tachi/mihon : Open Webview. Website will perform some checks and you should see manga page. Close webview and retry. :shrug:

**Comment by Karutovn** [comment]
> I just tested this workaround on Hakuneko, may work on tachi/mihon : Open Webview. Website will perform some checks and you should see manga page. Close webview and retry. :shrug:

Yeah I try it but it not work for me (it have some Annoying for me)
I think tachiyomi have update and adding library for bypassing devtool and ajax (not using okhttps)
